digitalocean/app/app_spec.go

```diff
@@ -189,6 +188,40 @@ func appSpecAppLevelAlerts() *schema.Resource {
 				Default:  false,
 				Optional: true,
 			},
+			"notifications": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"email": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type:         schema.TypeString,
+								ValidateFunc: validation.StringLenBetween(3, 100),
+							},
+						},
+						"slack": {
```

Comment:
For consistency with HTTP API

```suggestion
						"slack_webhooks": {
```